components/Interest/RegisterInterestSection.js

```diff
@@ -24,10 +25,10 @@ class RegisterInterestSection extends Component {
   // When component mounts, make initial API call.
   // TODO do we need to change this to getInitialProps?
   async componentDidMount () {
-    const op = this.props.op
+    const op = this.props.op._id
     const me = this.props.meID
     try {
-      await this.props.dispatch(reduxApi.actions.interests.get({ id: '', op, me }))
+      await this.props.dispatch(reduxApi.actions.interests.get({ op, me, z: cuid() }))
```

Comment:
We need to show why we are adding the z: cuid() query parameter. Otherwise someone might think it is not being used and remove it.  Probably renaming z to cacheBreak: cuid()  would be sufficient.

Reply:
Ok! I will add some comments on it.